wiki/Community/osu!dev_Discord_server/en.md

```diff
@@ -5,29 +5,58 @@ tags:
 
 # osu!dev Discord server
 
-**osu!dev** is the official [Discord](https://discordapp.com) server for working on osu!'s open source and community projects. This is the main place to contribute to osu!'s development and get in touch with the [osu! team](/wiki/People/osu!_team).
+**osu!dev** is osu!'s official [Discord](https://discord.com/) server for development-related purposes. The server functions as a hub for people to discuss and work on osu!'s open source and community projects. This is the main place to contribute to osu!'s development and get in touch with the [osu! team](/wiki/People/osu!_team).
 
-The permanent invite link is <https://discord.gg/ppy>.
+The permanent invite link for the server is <https://discord.gg/ppy>.
 
-People new to the server may feel free to introduce themselves in the `#introductions` channel, but doing so is not a requirement to interact with the rest of the server.
+People new to the server are encouraged to introduce themselves in the `#introductions` channel, but doing so is not a requirement to interact with the rest of the server.
 
-## Projects
+## Channels
 
-Each main development discussed in the server has its own channel:
+The osu!dev Discord server features a number of channels, many of which are centered around the development of a specific project or two.
 
-| Project | Channel |
+### General
+
+| Channel | Project |
+| :-- | :-- |
+| `#welcome` | *N/A* |
+| `#introduction` | *N/A* |
+| `#general` | *N/A* |
+
+### Client / Web
+
+| Channel | Project |
+| :-- | :-- |
+| `#osu` | [osu!(lazer)](/wiki/Client/Release_stream/Lazer) |
+| `#osu-framework` | [osu!framework](https://github.com/ppy/osu-framework) |
+| `#osu-web` | [osu!web](https://github.com/ppy/osu-web) |
+| `#github-feed` | *N/A* |
+| `Meetings` (Stage channel) | [osu! community meetings](/wiki/Community/osu!_community_meetings) |
+
+### Diffcalc
+
+| Channel | Project |
+| :-- | :-- |
+| `#difficulty-osu` | [osu!](/wiki/Game_mode/osu!) [star rating](/wiki/Beatmap/Star_rating) and [performance points](/wiki/Performance_points) calculation |
+| `#difficulty-taiko` | [osu!taiko](/wiki/Game_mode/osu!taiko) star rating and performance points calculation |
+| `#difficulty-catch` | [osu!catch](/wiki/Game_mode/osu!catch) star rating and performance points calculation |
+| `#difficulty-mania` | [osu!mania](/wiki/Game_mode/osu!mania) star rating and performance points calculation |
+
+### Community
+
+| Channel | Project |
+| :-- | :-- |
+| `#featured-artists` | [Featured Artist](/wiki/People/Featured_Artists) beatmap creation |
+| `#mappers-guild` | The [Mappers' Guild](/wiki/Community/Mappers_Guild) |
+| `#osu-spotlights` | [Beatmap Spotlights](/wiki/Beatmap_Spotlights) |
+| `#osu-wiki` | [osu! wiki](/wiki/osu!_wiki) |
+| `#osu-news` | [osu!monthly](/wiki/Community/osu!monthly) |
```

Comment:
```suggestion
| `#osu-news` | [osu!monthly](/wiki/Community/osu!monthly) |
```
also used for general newspost stuff, though monthly seems to be the main purpose, not sure how to handle this